data.sql

```diff
@@ -1,6 +1,16 @@
 /* Populate database with sample data. */
 
+/* Project 1 */
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '02-03-2020', 0, TRUE, 10.23); 
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon',  '15-11-2018', 2, TRUE, 8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon',  '15-11-2018', 2, TRUE, 8.0);
```

Comment:
- [ ] Please make sure that all your dates and times follow the correct PostgreSQL format because we getting errors and PostgreSQL is **yyyy-mm-dd**. Note that this change could affect your queries so make sure that everything is running fine :smile: 
 
![Screenshot from 2022-10-18 22-59-47](https://user-images.githubusercontent.com/6615394/196560943-6c86302e-daad-49c6-96ee-b03de0304012.png)

Reply:
Ohhh ok, thank you very much for the recommendation. On my computer, the SQL was ordering it as you told me automatically, that's why I didn't realize it but I already corrected it and checked the queries and there was no problem. 😃 

![image](https://user-images.githubusercontent.com/106561762/196578188-2a5e8aaf-c730-4766-a017-06f5e7b05d85.png)